charts/blackduck-connector/values.yaml

```diff
@@ -0,0 +1,153 @@
+# Default values for Opssight.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+nameOverride: ""
+fullnameOverride: ""
+
+# Docker registry to pull Black Duck images
+registry: docker.io/blackducksoftware
+imageTag: 2.2.5
+
+# image pull secret to download the Black Duck images (mostly applicable for air gapped customers)
+imagePullSecrets: [] # array of strings delimited by comma
+
+# This is used to start or stop the app. Set to Running to start, or Stopped to stop.
+status: Running
+
+logLevel: "debug"
+
+# External Hub ~>
+#  {
+#   "<BLACK_DUCK_URL>": 
+#     {
+#         "scheme":"https",
+#         "domain":"<BLACK_DUCK_URL>",
+#         "port":443,
+#         "user":"<BLACK_DUCK_USERNAME>",
+#         "password":"<BLACK_DUCK_PASSWORD>",
+#         "concurrentScanLimit":3
+#     }
+#   }
+
+# Secured Registries ~> 
+# {
```

Comment:
Similarly secured registries is an array

Reply:
I think operator kinda did that conversion from array to map internally, cos I tried array and it won't work. It needs a map